lib/ansible/modules/system/java_cert.py

```diff
@@ -334,10 +322,18 @@ def main():
 
     if state == 'absent':
```

Comment:
I'd collapse if blocks to save on nesting +1 to readability:
```suggestion
    if state == 'absent' and cert_present:
```